docs/aws-lambda-deploy.md

```diff
@@ -0,0 +1,9 @@
+---
+id: aws-lambda-deploy
+title: Deploy PayID with AWS Lambda
+sidebar_label: Deploy PayID with AWS Lambda
+---
+
+**NOTES**
+Skeleton doc.
+Actual doc is in review.
```

Comment:
This is fine for now, but I don't recommend merging placeholders like this to the main branch. In practice, I think most placeholders or "coming soon" messages end up sitting around indefinitely until they're removed without actually being implemented.

Reply:
I will get all the content in before merging, and I won't leave any placeholders. I prefer to omit the topic entirely if I had to (and this doc does exist now).